client/app/private-database/configuration/private-database-configuration.html

```diff
@@ -1,76 +1,61 @@
-                                  data-tooltip-append-to-body="true">
-                            </span> :
-                        </label>
-                        <div class="col-md-8">
-                            <span class="d-block form-control-static"
-                                  data-ng-if="!configurationsCtrl.edit.value">
-                                <span data-ng-bind="configuration.value"></span>
-                                <span data-i18n-static="{{'unit_configuration_' + configuration.unit}}" data-ng-if="configuration.unit"></span>
-                            </span>
-                            <div class="oui-select oui-select_inline" data-ng-if="configurationsCtrl.edit.value">
-                                <select class="oui-select__input" name="{{configuration.key}}"
-                                        data-ng-attr-id="{{configuration.key + 'Field'}}"
-                                        data-ng-model="configuration.value"
-                                        data-ng-options="conf for conf in configuration.availableValues">
-                                </select>
-                                <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
-                            </div>
-                        </div>
-                    </div>
-                </form>
-            </div>
-        </div>
+            <p data-i18n-static="privateDatabase_configuration_alert"></p>
 
-        <div class="col-md-3 mt-5 mt-lg-0"
-             data-ng-if="configurationsCtrl.database.type !== 'postgresql'">
-            <button class="btn btn-block btn-default" type="button"
-                    data-i18n-static="privateDatabase_configuration_edit_btn"
-                    data-ng-click="configurationsCtrl.edit.value = true"
-                    data-ng-disabled="database.expired || isRebooting"
-                    data-ng-if="!configurationsCtrl.edit.value">
-            </button>
-            <button class="btn btn-block btn-primary" type="button"
-                    data-i18n-static="privateDatabase_configuration_confirm_btn"
-                    data-ng-click="configurationsCtrl.updateConfigurations()"
-                    data-ng-disabled="database.expired"
-                    data-ng-if="configurationsCtrl.edit.value">
-            </button>
-            <button class="btn btn-block btn-default" type="button"
-                    data-i18n-static="privateDatabase_configuration_cancel_btn"
-                    data-ng-click="configurationsCtrl.configurations = configurationsCtrl.baseConfiguration; configurationsCtrl.edit.value = false"
-                    data-ng-disabled="database.expired"
-                    data-ng-if="configurationsCtrl.edit.value">
-            </button>
+            <form name="configurationsCtrl.dbConfigurationForm"
+                  data-ng-submit="configurationsCtrl.dbConfigurationForm.$valid && configurationsCtrl.updateConfigurations()"
```

Comment:
I don't know either, but @tremblaymath ask me to, I guess he have a better explanation then what I recall that also explain the  ```novalidate```

Reply:
novalidate is used to disable browser's native form validation, we don't want it. https://scotch.io/tutorials/angularjs-form-validation#toc-disabling-html5-validation-novalidate
ng-submit is triggered even if the form is invalid, you can test it there:
https://plnkr.co/edit/o5CHGvNRwNHH7105ndcb?p=preview
ng-submit is flexible because we can handle invalid forms in the controller
@JDownloader it's important to know why you add code, ask or test it

Reply:
@tremblaymath thanks for your feedback :)

I understand what is `novalidate` attribute, but i don't understand why you use it if you add an custom equivalent in the submit event.

It seems complicated for nothing, why don't use the native form validation ? If the form is invalid, no need to submit it, right ?

Is it a CX requirements ? We can submit a invalid form ?

Reply:
Hi @jleveugle, native form validation are ugly, we don't want it, we want full control
<img width="378" alt="capture d ecran 2018-04-17 a 07 56 47" src="https://user-images.githubusercontent.com/10472038/38868424-f8893166-4214-11e8-8f39-776ddeab008c.png">
noValidate is used in angularjs doc as their first form example https://docs.angularjs.org/guide/forms
It's a cx requirement to always be able to submit an invalid form. Submit button is always available.
Have you something better to propose? Could you edit that plunker to show me please https://plnkr.co/edit/o5CHGvNRwNHH7105ndcb?p=preview